pyspectre/core.py

```diff
@@ -2,12 +2,14 @@
 
 import os
 import re
+import subprocess
 from   subprocess      import run, DEVNULL, Popen
 from   tempfile        import NamedTemporaryFile
 import errno
 import warnings
 from   collections.abc import Iterable
-from   typing          import NamedTuple, NewType
+from   typing          import NamedTuple, NewType, List, Dict, Iterable
```

Comment:
`NamedTuple` can be removed and `Iterable` now clashes with the import above. I suggest removing Line 10.